com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraUI.Drawers.cs.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
```

Comment:
This should be moved back to com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs and appropiate one needs to be generated for urp